english/hulk/stop-hulk.md

```diff
@@ -0,0 +1 @@
+You rocket into the sky and head straight for Gotham. You arrive to find half of the city completely destroyed. You are just in time to watch the Hulk kill Batman. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.